docs/package.json

```diff
@@ -22,7 +22,6 @@
     "@storybook/addon-essentials": "^6.5.12",
     "@storybook/addon-interactions": "^6.5.12",
     "@storybook/addon-links": "^6.5.12",
-    "@storybook/addon-storyshots": "^6.5.15",
     "@storybook/addon-storysource": "^6.5.12",
     "@storybook/builder-vite": "^0.2.2",
     "@storybook/jest": "^0.0.10",
```

Comment:
this is also unused, AFAICT, just imported once in `docs/src/components/TextSearch.stories.tsx`